src/Schema/Directives/RenameDirective.php

```diff
@@ -38,22 +38,36 @@ public static function definition(): string
      * @param  \Nuwave\Lighthouse\Schema\Values\FieldValue  $fieldValue
      * @return \Nuwave\Lighthouse\Schema\Values\FieldValue
      *
-     * @throws \Nuwave\Lighthouse\Exceptions\DirectiveException
+     * @throws \Nuwave\Lighthouse\Exceptions\DefinitionException
```

Comment:
That annotation has to go, we only annotate in the exact method where exceptions are thrown.